subsystem/kubeapi_test.go

```diff
@@ -342,6 +342,46 @@ var _ = Describe("[kube-api]cluster installation", func() {
 		}, "2m", "10s").Should(Equal(validHwInfo.SystemVendor.Manufacturer))
 	})
 
+	It("deploy clusterDeployment and installEnv and validate image generated and image type", func() {
```

Comment:
why not test it in already existing test `deploy clusterDeployment and installEnv and verify cluster updates"` ?

Reply:
I thought about it. wondered if we wanted to have a "basic" test without any additional params.
I can place this in the test that you've mentioned

Reply:
you are just validating one parameter in the DB, you case place it in a `By` section.

Reply:
will do

Reply:
updated:
```sh
[kube-api]cluster installation
  deploy clusterDeployment and installEnv and verify cluster updates
  /home/nmagnezi/go/src/github.com/assisted-service/subsystem/kubeapi_test.go:345
STEP: Validate proxy settings.
STEP: Validate additional NTP settings.
STEP: InstallEnv image type defaults to minimal-iso.
```